mk/config.mk

```diff
@@ -38,8 +38,10 @@ WARNS ?= 3
 # Define DEBUG=1 to compile with -g option
 # DEBUG=1
 
-# If 1, debug mode of the tee firmware (CPU restart, Core Status)
-CFG_TEE_CORE_DEBUG ?= 0
+# Core debug mode enablement
+# When defined to "y", tee firmware is in debug mode. Core and pager status,
+# panics verbosity, physical/virtual address paranoid verifications.
+CFG_TEE_CORE_DEBUG ?= n
```

Comment:
I wouldn't mind having that changed. But it would be better to take in a separate PR which I assume could be merged before this PR.